src/sites/amopis/pages/AmopisRootPage.tsx

```diff
@@ -1,19 +1,20 @@
+import '@amsterdam/design-system-tokens/dist/compact.css'
```

Comment:
This is not the way to include the compact tokens – as soon as you’ve opened the page, the other prototypes will use them too. @alimpens Do you have a suggestion how to do this instead?

Reply:
Ah, I think you've found a use case for the `*.theme.css` files. When you import a css file, it remains in the local assets of a site, it is not removed on page change or anything. So when you have a site that uses multiple modes (like this one, and probable all dark mode implementations) you have to toggle a class instead of just loading all vars in `:root`. 

The classes should be unique though, currently they're all called `amsterdam-theme`. What do you think of `amsterdam-theme--compact`?

Reply:
I'll make a quick PR for that in ADS